test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -35,6 +35,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework/providers/gce"
 	e2eresource "k8s.io/kubernetes/test/e2e/framework/resource"
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	manifestutils "k8s.io/kubernetes/test/e2e/manifest"
```

Comment:
How about `e2emanifest` here like the above?

Reply:
manifestutils is also ok , but i will update it . thanks

Reply:
done

Reply:
/cc @oomichi 
```
Up to now we have been using the e2e prefix for framework subpackages.
I.e., if it the package was in k8s.io/kubernetes/test/e2e/framework/manifest then that one would be e2emanifest
```
That sounds good. how about manifestutils? I think it is better currently.

Reply:
I don't think manifestutils is better than e2e manifest, because all e2e test subpackage is named as "e2e" + "<subpackage name>" like https://github.com/kubernetes/kubernetes/pull/89454/files
Why do you want to name this on different way?

Reply:
humm, I need to think this again.
It is fine to move a function from `e2e/framework/util.go` into specific e2e test package if the function is used at the test package only.
However, this makes another dependency from 2 different test packages (e2e/scheduling and e2e_node) without e2e test framework which is shared place for different test packages. That is different from the existing way, is that intentional?

/lgtm cancel

Reply:
yes , you are right. It is different from the existing way. 
But  I  think it is  good to  maintain *FromURL functions in test/e2e/manitest package . They are the same or  homologous function.

Reply:
Humm, I am not sure this direction is valid for us.
e2e test framework should be a single place to hold shared functions between multiple e2e test packages, I did think before reviewing this PR.
If going this PR direction, I am not sure what is e2e test framework is.
I'd like to see more opinions about this.

/hold

Reply:
we can make an argument for moving that method outside of the e2e framework as the framework is not only mean to hold shared functions between e2e packages but it is meant to hold useful functions for e2e tests. Instantiating a daemonset from a manifest that can be download from a URL is not something to have in the e2e framework.
As is, it is good to move it out. To keep it in the framework (one of its subpackages) we should rewrite it to create any sort of Kubernetes resource based on a manifest that can be specified by a URL.